jetty-webapp/src/main/java/org/eclipse/jetty/webapp/Configurations.java

```
@@ -74,27 +75,18 @@
     {
         if (__known.isEmpty())
         {
-            ServiceLoader<Configuration> configs = ServiceLoader.load(Configuration.class);
-            for (Iterator<Configuration> i = configs.iterator(); i.hasNext(); )
+            TypeUtil.serviceStream(ServiceLoader.load(Configuration.class)).forEach(configuration ->
```

Again behavior change, do we really want to ignore exceptions here?